aliases.go

```diff
@@ -49,41 +49,23 @@ func (client *Client) CreateAlias(input AliasCreateInput) ([]string, error) {
 	return output, HandleErrors(err, m.Payload.Errors)
 }
 
-func (client *Client) DeleteDomainAlias(alias string) error {
-	return client.DeleteAlias(AliasDeleteInput{
-		Alias:     alias,
-		OwnerType: AliasOwnerTypeEnumDomain,
-	})
-}
-
+// Deprecated: use client.DeleteAlias instead
 func (client *Client) DeleteInfraAlias(alias string) error {
 	return client.DeleteAlias(AliasDeleteInput{
 		Alias:     alias,
 		OwnerType: AliasOwnerTypeEnumInfrastructureResource,
 	})
 }
 
-func (client *Client) DeleteScorecardAlias(alias string) error {
-	return client.DeleteAlias(AliasDeleteInput{
-		Alias:     alias,
-		OwnerType: AliasOwnerTypeEnumScorecard,
-	})
-}
-
+// Deprecated: use client.DeleteAlias instead
 func (client *Client) DeleteServiceAlias(alias string) error {
 	return client.DeleteAlias(AliasDeleteInput{
 		Alias:     alias,
 		OwnerType: AliasOwnerTypeEnumService,
 	})
 }
 
-func (client *Client) DeleteSystemAlias(alias string) error {
```

Comment:
This was added yesterday, removing today. We don't want to propagate this pattern.